modules/cluster-cloud-controller-config-osp.adoc

```diff
@@ -0,0 +1,272 @@
+
+| `cloud`
+| The named cloud in the `clouds.yaml` file that you want to use. It is used if the `use-clouds` option is set to `true`.
+
+| `application-credential-id`
+| The ID of an application credential to authenticate with. An `application-credential-secret` must be set along with this parameter.
+
+| `application-credential-name`
+| The name of an application credential to authenticate with. If `application-credential-id` is not set, the user name and domain must be set.
+
+| `application-credential-secret`
+| The secret of an application credential to authenticate with.
+
+| `tls-insecure`
+| Whether or not to verify the server's TLS certificate. If set to `true`, the certificate is not verified. By default, the certificate is verified.
+|===
+
+
+// [id="ccm-config-networking-options"]
+// == Networking options
+
+// |===
+// | Option | Description
+
+// | `ipv6-support-disabled`
+// | Whether or not IPv6 is supported as indicated by a boolean value. By default, this option is `false`.
+
+// | `public-network-name`
+// | The name of an {rh-openstack} Networking service, or Neutron, external network. The CCM uses this option when retrieving the external IP address of a Kubernetes node. This value can contain multiple names. Specified networks are bitwise ORed. The default value is `""`.
+
+// | `internal-network-name`
+// | The name of a Networking service internal network. The CCM uses this option when retrieving the internal IP address of a Kubernetes node. This value can contain multiple names. Specified networks are bitwise ORed. The default value is `""`.
+
+// | `address-sort-order`
+// | This configuration key affects how the provider reports node addresses to Kubernetes node resources. The default order depends on the hard-coded order in which the provider queries addresses and what the cloud returns. A specific order is not guaranteed.
+
+//  To override this behavior, specify a comma-separated list of CIDR addresses. CCM sorts and groups all addresses that match the list in a prioritized manner, wherein the first retrieved item has  a higher priority than the last. Addresses that do not match the list remain in their default order. The default value is `""`.
+
+// This option can be useful if you have multiple or dual-stack interfaces attached to a node that need a user-controlled, deterministic way of sorting addresses.
+// |===
+
+[id="ccm-config-lb-options"]
+== Load balancer options
+
+CCM supports several load balancer options for deployments that use Octavia. 
+
+[NOTE]
+====
+Neutron-LBaaS support is deprecated. 
+====
+
+// Commented options are untested as of yet.
+
+|===
+| Option | Description
+
+| `enabled`
+| Whether or not to enable the `LoadBalancer` type of services integration. The default value is `true`.
+
+| `use-octavia`
+| Whether or not to use Octavia for the `LoadBalancer` type of service implementation rather than Neutron-LBaaS. The default value is `true`.
```

Comment:
```suggestion
// Removed: always enforced.
//| `use-octavia`
//| Whether or not to use Octavia for the `LoadBalancer` type of service implementation rather than Neutron-LBaaS. The default value is `true`.
```